<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>apply_filters</class>
 <widget class="QWidget" name="apply_filters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Apply Filters</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/FaceAttend2.png</normaloff>assets/FaceAttend2.png</iconset>
  </property>
  <layout class="QGridLayout" name="grid_layout">
   <item row="8" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="label_filter_icon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>assets/filter.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_select">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>assets/select.png</normaloff>assets/select.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_spacer_bottom">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="grid_layout_1">
     <item row="0" column="0" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="button_back">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/return.png</normaloff>assets/return.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_update_attendance">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#888a85;&quot;&gt;APPLY FILTERS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_spacer">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_placeholder">
     <property name="text">
      <string>Select the subject and the corresponding date for which you want to update the record:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <layout class="QGridLayout" name="grid_layout_2">
     <item row="1" column="2">
      <widget class="QComboBox" name="combo_box_date"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_subject">
       <property name="text">
        <string>Subject:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_box_subject"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_spacer_right">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_spacer_left">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_spacer_top">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_select</tabstop>
  <tabstop>button_back</tabstop>
 </tabstops>
 <resources>
  <include location="FaceAttend.qrc"/>
 </resources>
 <connections/>
</ui>
